components/HomeScreen.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { Text, View } from 'react-native';
+import { MaterialTopTabNavigationProp } from '@react-navigation/material-top-tabs';
+import { RootNavParamList } from '../App';
+
+import getEnvVars from '../environment';
+
+const { message } = getEnvVars();
+
+type HomeScreenNavigationProp = MaterialTopTabNavigationProp<
+  RootNavParamList,
+  'Home'
+>;
+
+type HomeScreenProps = {
+  navigation: HomeScreenNavigationProp;
```

Comment:
If it's a requirement to add these parameters to the props interface, consider using intersection types instead of defining a whole extra type for the navigation https://www.typescriptlang.org/docs/handbook/advanced-types.html#intersection-types
In general, though, if it's not used, I'd rather we not add it. Unless it doesn't work otherwise

Reply:
Looks like it isn't actually necessary unless we want to be able to use the navigation object from within the screen components. I'll remove.